src/m365/spo/commands/site/site-chrome-set.ts

```diff
@@ -0,0 +1,179 @@
+import { Logger } from '../../../../cli';
+import {
+  CommandOption
+} from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import SpoCommand from '../../../base/SpoCommand';
+import commands from '../../commands';
+
+export enum HeaderLayout {
```

Comment:
Since these enums are not used outside of this file, I don't think we need to export them, right?